server/src/controllers/coursera/getCourses.js

```diff
@@ -0,0 +1,86 @@
+const axios = require('axios');
+const courseraInsert = require('../../database/queries/insertCourses');
```

Comment:
in database/query/index you need to make it start point to get the query like this
https://github.com/GSG-G8/courses-search/blob/73664eceb2db094f07bae30aa204246b690c8520/server/src/database/queries/index.js#L1-L5
and here call just the query folder
```suggestion
const courseraInsert = require('../../database/queries');
```